contrib/slime/README.org

```diff
@@ -74,6 +74,15 @@ of Common Lisp, you can specify it in your =~/.spacemacs=
 | ~SPC m c f~ | Compile function         |
 | ~SPC m c r~ | Compile region           |
 
+** Definition
+
+| Key Binding               | Description                     |
+|---------------------------+---------------------------------|
+| ~SPC m d d~               | Go to definition                |
+| ~SPC m d u~ or ~SPC m d D | Return from definition          |
+| ~SPC m d w~               | Open definition in other window |
+| ~SPC m d f~               | Open definition in other frame  |
```

Comment:
I would change it for `SPC m g C-g` since it should not be used so often and it is still in the same group of functionality under `m g g`.